faq/FAQ_Formatting_Guide.txt

```diff
@@ -399,7 +399,7 @@ they are the final editors of the text.
 Because of this, post-processing is only available for proofreaders who have completed at least
 <b>400</b> pages in the first and/or second proofreading rounds. 
 Also, post-processors should be very familiar with the
-<a href="document.php">Proofreading Guidelines</a> before attempting to post-process.
+<a href="proofreading_guidelines.php">Proofreading Guidelines</a> before attempting to post-process.
 </p>
```

Comment:
This may need to be rewritten, too. document.php was mapped to formatting)guidelines.php, not proofreading_guidelines.php.